src/components/tooltip/tooltip.js

```diff
@@ -406,9 +441,9 @@ function MdTooltipRegistry() {
   };
 
   /**
-   * Global event handler that dispatches the registered
-   * handlers in the service.
-   * @param  {Event} event Event object passed in by the browser.
+   * Global event handler that dispatches the registered handlers in the
+   * service.
+   * @param {Event} event Event object passed in by the browser
```

Comment:
As per Erin's feedback last time, I learned that JSDoc like this should be marked as not-nullable

```
{!Event}
```

Same for below?